recipes/seleniumgrid.rb

```diff
@@ -0,0 +1,116 @@
+#
+# Cookbook Name:: aet
+# Recipe:: seleniumgrid
+#
+# AET Cookbook
+#
+# Copyright (C) 2016 Cognifide Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Create dedicated group
+group node['aet']['seleniumgrid']['group'] do
+  action :create
+end
+
+# Create dedicated user
+user node['aet']['seleniumgrid']['user'] do
+  group node['aet']['seleniumgrid']['group']
+  manage_home true
+  home node['aet']['seleniumgrid']['root_dir']
+  system true
+  shell '/bin/bash'
+  action :create
+end
+
+# Create dedicated hub directory
+directory "#{node['aet']['seleniumgrid']['root_dir']}/hub" do
+  owner node['aet']['seleniumgrid']['user']
+  group node['aet']['seleniumgrid']['group']
+  mode '0755'
+  action :create
+  recursive true
+end
+
+# Create dedicated node directory
+directory "#{node['aet']['seleniumgrid']['root_dir']}/ff" do
+  owner node['aet']['seleniumgrid']['user']
+  group node['aet']['seleniumgrid']['group']
+  mode '0755'
+  action :create
+  recursive true
+end
+
+# Create log directory
+directory "#{node['aet']['seleniumgrid']['log_dir']}" do
+  owner node['aet']['seleniumgrid']['user']
+  group node['aet']['seleniumgrid']['group']
+  mode '0755'
+  action :create
+  recursive true
+end
+
+# Get Selenium Grid file name from link
+filename = get_filename(node['aet']['seleniumgrid']['source'])
+
+# Download Selenium Grid jar
+remote_file "#{node['aet']['seleniumgrid']['root_dir']}/hub/#{filename}" do
+  owner node['aet']['seleniumgrid']['user']
+  group node['aet']['seleniumgrid']['group']
+  mode '0644'
+  source node['aet']['seleniumgrid']['source']
+end
+
+# Download Selenium Grid jar
+remote_file "#{node['aet']['seleniumgrid']['root_dir']}/ff/#{filename}" do
+  owner node['aet']['seleniumgrid']['user']
+  group node['aet']['seleniumgrid']['group']
+  mode '0644'
+  source node['aet']['seleniumgrid']['source']
+end
+
+# Create Selenium Grid hub init file
+template '/etc/init.d/seleniumgrid-hub' do
+  source 'etc/init.d/seleniumgrid-hub.erb'
+  owner 'root'
+  group 'root'
+  cookbook node['aet']['seleniumgrid']['src_cookbook']['init_script']
+  mode '0755'
+
+  notifies :restart, 'service[seleniumgrid-hub]', :delayed
+end
+
+# Enable and start Selenium Grid hub
+service 'seleniumgrid-hub' do
+  supports status: true, restart: true
+  action [:start, :enable]
+end
+
+# Create Selenium Grid FF node init file
+template '/etc/init.d/seleniumgrid-ff' do
```

Comment:
Could we change the suffix to `firefox-node`? So that it is really obvious what does it do?
I mean: someone then could try to find in goole something about *seleniumgrid* and *node* and have some starting point...